deps/Makefile

```diff
@@ -243,8 +243,8 @@ mariadb_client: mariadb-client-library/mariadb_client/libmariadb/libmariadbclien
 sqlite3/sqlite3/sqlite3.o:
 	cd sqlite3 && rm -rf sqlite-amalgamation-*/ || true
 	cd sqlite3 && tar -zxf sqlite-amalgamation-*.tar.gz
-	cd sqlite3/sqlite3 && patch -p1 < ../from_unixtime.patch
-	cd sqlite3/sqlite3 && patch sqlite3.c < ../sqlite3.c-multiplication-overflow.patch
+	cd sqlite3 && ln -fsT sqlite-amalgamation-*/ sqlite3
```

Comment:
I am against creating new symlink while Makefile is running.
Symlinks should already exist .